core/lib/server/pub.db.test.ts

```diff
@@ -793,8 +644,8 @@ describe("getPubsWithRelatedValuesAndChildren", () => {
 			}
 		);
 
-		// 3 root pubs, one of which has 1 child pub & 1 related pub, even though the limit is 3. This is correct behavior.
-		expect(pubs.length).toBe(5);
+		// 3 root pubs, one of which 1 related pub, even though the limit is 3. This is correct behavior.
+		expect(pubs.length).toBe(3);
```

Comment:
hmm why wouldn't this be `4`?

Reply:
I think the comment may be wrong? There are in fact only 3 pubs in the community, but I may need to update the test to preserve its intent... I'll look into it!